doc/naming.md

```diff
@@ -41,6 +41,12 @@ Most gRPC implementations support the following URI schemes:
     three slashes, two prior to the path and another to begin the
     absolute path).
 
+- `unix:@abstract_unix_socket` -- Abstract Unix domain sockets (Linux systems only)
```

Comment:
Rather than adding this as a separate entry, please update the exist entry for the `unix:` scheme.